### DISCUSSION TITLE
<title>Ocorreu um erro durante a instalação do GitHub Desktop. Como posso corrigir isso?</title>

### DISCUSSION THREAD

**Discussion by SUM-SuporteUnionMundial** [community/community#62928] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

"Olá,

Estou tendo problemas para instalar o GitHub Desktop. Quando tento instalá-lo, recebo a mensagem de erro "Ocorreu um erro durante a instalação do aplicativo. Verifique o log de configuração para obter mais informações e entre em contato com o autor."

Já tentei limpar a instalação e executar algumas ferramentas de reparo do Windows, mas não consegui resolver o problema.

Alguém pode me ajudar?

Atividades; Gostaria de dizer que ja instalei em outro momentos o github desktop no windows 8.1 e funcionou normalmente porem agora no windows 10 esta dando erro, tentei instalar uma maquina virtual com windows 8.1 pelo virtual box mas tambem não deu certo.
Ja tentei limpar as pastas temporarias que aparecem durante a criação para tentar reinstalar, tentei executar o instalado pelo cmd em modo ADM mas , não obtive exito .

**Comment by mehulrajdev** [comment]
Check this, please

[https://github.com/mehulrajdev/Zayn-Malik](https://github.com/mehulrajdev/Zayn-Malik)](https://github.com/mehulrajdev/Zayn-Malik%5D(https://github.com/mehulrajdev/Zayn-Malik))

https://github.com/mehulrajdev
